### ISSUE TITLE
Broken in Home Assistant 2021.6

### ISSUE THREAD

**Issue by SquaredCircleHunter** [ljmerza/light-entity-card#65]
Hi,

The main 'lit-element' module entrypoint is deprecated. Please update your imports to use the 'lit' package: 'lit' and 'lit/decorators.ts' or import from 'lit-element/lit-element.ts'.

https://developers.home-assistant.io/blog/2021/05/19/lit-2.0

**Comment by ljmerza** [comment]
The card has always worked for me as I do not use HA's internal lit but my own rolled up version.

**Comment by SquaredCircleHunter** [comment]
I copy pasted the output in the browser console log. Didn't dive into the technical details, sounds like it's a false positive then. My card stopped working at the time of posting, maybe it's fixed in the mran time

**Comment by johnatanfeuillye** [comment]
Hi @ljmerza,
I'm also having same issue as @SquaredCircleHunter, where the card is no longer working since 2016.6
I have re-installed multiple times via hacs and manually with no success.
Would be great to see your card back in my HA :)

**Comment by ljmerza** [comment]
are there any errors in the console? What's your config look like?

**Comment by justr0st** [comment]
Same issue with HA 2021.7.0
For example:
this is a light entity, that allows selecting color, brightness and effect. Only effect is visible.
![config](https://user-images.githubusercontent.com/50486502/124929636-f467d780-e000-11eb-8268-ae0661939349.PNG)

**Lovelace tab config:**
  - cards:
      - type: custom:light-entity-card
        shorten_cards: false
        consolidate_entities: false
        color_wheel: true
        persist_features: false
        brightness: true
        color_temp: false
        white_value: true
        color_picker: true
        smooth_color_wheel: true
        hide_header: false
        child_card: false
        show_slider_percent: false
        full_width_sliders: true
        brightness_icon: weather-sunny
        white_icon: file-word-box
        temperature_icon: thermometer
        header: Office_ambilight
        effects_list: true
        entity: light.office_ambilight

**Browser console log:**

app.62541dd1.js:13499 The main 'lit-element' module entrypoint is deprecated. Please update your imports to use the 'lit' package: 'lit' and 'lit/decorators.ts' or import from 'lit-element/lit-element.ts'.
55704 @ app.62541dd1.js:13499
s @ app.62541dd1.js:36082
53918 @ app.62541dd1.js:1279
s @ app.62541dd1.js:36082
(anonymous) @ app.62541dd1.js:36082
(anonymous) @ app.62541dd1.js:42188
(anonymous) @ app.62541dd1.js:42188
browser_mod.js:1 Event {isTrusted: false, detail: "connected", type: "connection-status", target: Window, currentTarget: Window, …}
ha-card-weather-conditions.js:381  WEATHER-CONDITION-CARD  1.9.9
light-entity-card.js:9160   LIGHT-ENTITY-CARD     Version 4.2.2       
mini-graph-card-bundle.js:1  MINI-GRAPH-CARD  0.10.0 
buien-rain-card.js:309 BUIEN-RAIN-CARDVersion: 0.0.4
card-mod.js:5 CARD-MOD 3.0.12 IS INSTALLED 
button-card.js:425   BUTTON-CARD   Version 3.4.2 
floorplan.js:87 Floorplan for Home Assistant (ha-floorplan)Version 1.0.25
slider-button-card.js?hacstag=3754399611103:497   SLIDER-BUTTON-CARD  v1.10.3 
entities-btn-group.js?hacstag=373857882102:186   Entities-Button-Group   Version 1.0.2    
cloudy.svg:1 GET http://192.168.1.100:8123/local/ha-card-weather-conditions/icons/static/cloudy.svg 404 (Not Found)
Image (async)
(anonymous) @ ha-card-weather-conditions.js:270
setTimeout (async)
(anonymous) @ ha-card-weather-conditions.js:270
Ct @ ha-card-weather-conditions.js:270
(anonymous) @ ha-card-weather-conditions.js:381
ha-card-weather-conditions.js:381  WEATHER-CONDITION-CARD  1.9.9 use HACS path to retrieve icons.
browser_mod.js:142 BROWSER_MOD 1.3.1 IS INSTALLED    DeviceID: f5259b79-1ffbd64d

**Comment by johnatanfeuillye** [comment]
Hi Ljmerza,
Sorry for the delay, this is what I find in my console browser when loading HA

![image](https://user-images.githubusercontent.com/54647850/125000183-29bd0580-e093-11eb-860b-e8ca3b469f55.png)

**Comment by ljmerza** [comment]
the light entiy card did load - you can see that from the black/grey box.something else is wrong

**Comment by SquaredCircleHunter** [comment]
@johnatanfeuillye @justr0st 

Are you using KNX lights?

**Comment by johnatanfeuillye** [comment]
Thanks @ljmerza for quick reply.  I'm not sure what it can be...

**Comment by johnatanfeuillye** [comment]
> @johnatanfeuillye @justr0st
> 
> Are you using KNX lights?

I'm not using KNX devices.  majority of my device are mqtt

**Comment by SquaredCircleHunter** [comment]
I plan RCA but can't do it very soon.

**Comment by johnatanfeuillye** [comment]
This morning I updated HA to 2021.7.1. Reinstalled light-card-entity and now it works...Any way, I'm happy now
![image](https://user-images.githubusercontent.com/54647850/125051941-cd83d100-e0e6-11eb-8f8f-722d3edb8dc7.png)

**Comment by SquaredCircleHunter** [comment]
Hi,

I just upgraded to Home Assistant 2021.7.1 and reinstalled the latest version of the card.

It is still not working.

I can confirm the LIT-thing is indeed completely unrelated.

What I found is that the color wheel is not being shown, because the dontShowFeature method requires "hs" being supported as a color mode: e.attributes.supported_color_modes && e.attributes.supported_color_modes.includes("hs");

`
The array contains only "rgb": e.attributes.supported_color_modes === ["rgb"]
`

When, for the sake of testing, I add "hs" to the supported_color_modes array in debug mode, the color wheel is shown.

I don't get it, because this worked in the past. My lights are RGB:

`
supported_color_modes:
  - rgb
supported_features: 0
`

My card config (I enabled persist_features, doesn't have effect)

`
          - shorten_cards: false
            consolidate_entities: false
            child_card: false
            hide_header: false
            header: ''
            color_wheel: true
            persist_features: true
            brightness: true
            color_temp: true
            white_value: true
            color_picker: true
            smooth_color_wheel: false
            show_slider_percent: false
            full_width_sliders: false
            brightness_icon: weather-sunny
            white_icon: file-word-box
            temperature_icon: thermometer
            type: custom:light-entity-card
            entity: light.toilet_rgbw
`

I am using KNX lights, my config did not change. It is theoretically possible that in the past they claimed support for "hs" and now not anymore. But the config is clearly RGB:

`
name: "XXXXXXXXXX"
  address: 'X/0/11'
  state_address: 'X/1/11'
  brightness_address: 'X/3/11'
  brightness_state_address: 'X/4/11'
  color_address: 'X/5/3'
  color_state_address: 'X/6/3'
`

https://www.home-assistant.io/integrations/knx/#light

(for my future reference, color modes: https://developers.home-assistant.io/docs/core/entity/light)

**Comment by ljmerza** [comment]
try 4.4.1

**Comment by SquaredCircleHunter** [comment]
I just tested 4.2.2, works again!

**Comment by justr0st** [comment]
> @justr0st The console error is not from my card, it looks to be from home assistant itself. but it looks like your card is working? I can see it in the picture.

@ljmerza Yes, the card was loaded and working, but without any controls shown, like color wheel and brightness. Only Effect selector was there.

However, with updated version 4.4.0 it works again. And here is what my original and unchanged card looks like now:
![config_new](https://user-images.githubusercontent.com/50486502/125540699-8e029625-76b8-461b-86be-8b2aa1b9bf15.PNG)
